utils/fetchIssuedAssets.ts

```diff
@@ -0,0 +1,60 @@
+import ERC725js, { ERC725JSONSchema } from '@erc725/erc725.js';
+import LSP12Schemas from '@erc725/erc725.js/schemas/LSP12IssuedAssets.json';
+import LSP4Schemas from '@erc725/erc725.js/schemas/LSP4DigitalAsset.json';
+
+const verifyCreatorShip = async (
+  ownerAddress: string,
+  provider: any,
```

Comment:
can you do better than `any` ? If not all good.

Reply:
It is a web3 httpProvider but web3.js does not export its type